scripts/hook_injector.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+'''
+hook_injector.py
+
+    This example harnesses the power of
+    Manticore as a debugger during program analysis,
+    providing introspection of hooks injected through
+    user-specified pattern inputs.
+
+'''
+import re
+import argparse
+
+from manticore import Manticore
```

Comment:
This will need to be changed to `from manticore.native import Manticore` after https://github.com/trailofbits/manticore/pull/1257 is merged (shall be soonish/today)